### DISCUSSION TITLE
Revit Lookup table row graphics

### DISCUSSION THREAD

**Discussion by IdoBand** [jeremytammik/RevitLookup#258] [Q&A] [answered]
Hi
Im trying to use Revit Lookup add-in to snoop elements and it seems that some rows looks like they are disabled, but they are not, they are functioning but this graphics are making it really hard to work with.
I think these are suppose to be Bold as seen in Eric’s Fritz ebook.
Is there a way to change that?

The only thing I managed to change was to enable dark mode but then all the rows has the exact same font, which isnt helpful either

![image](https://github.com/jeremytammik/RevitLookup/assets/117296781/7f89ca6f-df5c-465e-8eb7-e92860cd3ba7)

**Comment by Nice3point** [comment, marked as answer]
As you can see here https://github.com/jeremytammik/RevitLookup/releases/tag/2024.0.0 RevitLlookup accent colour synced with Windows accent colour. Try to set another colour in the Windows settings

**Reply by IdoBand** [reply]
Thank you!
